documentation/content/el/books/handbook/introduction/_index.adoc

```diff
@@ -62,7 +62,7 @@ endif::[]
 [[nutshell]]
 == Καλώς ήλθατε στο FreeBSD!
 
-Το FreeBSD είναι ένα λειτουργικό σύστημα βασισμένο στο 4.4BSD-Lite, το οποίο μπορεί να χρησιμοποιηθεί σε Η/Υ Intel (x86 και Itanium(R)), AMD64 και Sun UltraSPARC(R). Σε εξέλιξη βρίσκεται επίσης η διαδικασία μεταφοράς του FreeBSD σε άλλες αρχιτεκτονικές. Μπορείτε επίσης να διαβάσετε για <<history,την ιστορία του FreeBSD>>, ή για την <<relnotes,πιο πρόσφατη επίσημη έκδοση του>>. Εάν ενδιαφέρεστε να συμβάλλετε με κάποιο τρόπο στο Project (κώδικα, hardware, χρηματοδότηση), διαβάστε το άρθρο extref:{contributing}[Συνεισφέροντας στην Ανάπτυξη του FreeBSD].
+Το FreeBSD είναι ένα λειτουργικό σύστημα βασισμένο στο 4.4BSD-Lite, το οποίο μπορεί να χρησιμοποιηθεί σε Η/Υ Intel (x86 και Itanium(R)) και AMD64. Σε εξέλιξη βρίσκεται επίσης η διαδικασία μεταφοράς του FreeBSD σε άλλες αρχιτεκτονικές. Μπορείτε επίσης να διαβάσετε για <<history,την ιστορία του FreeBSD>>, ή για την <<relnotes,πιο πρόσφατη επίσημη έκδοση του>>. Εάν ενδιαφέρεστε να συμβάλλετε με κάποιο τρόπο στο Project (κώδικα, hardware, χρηματοδότηση), διαβάστε το άρθρο extref:{contributing}[Συνεισφέροντας στην Ανάπτυξη του FreeBSD].
```

Comment:
i don't know where that info is from, we don't support Ithanium any more: https://www.freebsd.org/platforms/

Reply:
Yes that's true. The content is too old. Was planning to remove it :)